instrumentation/google.golang.org/grpc/otelgrpc/stats_handler.go

```diff
@@ -121,7 +121,7 @@ func (h *clientHandler) TagRPC(ctx context.Context, info *stats.RPCTagInfo) cont
 
 // HandleRPC processes the RPC stats.
 func (h *clientHandler) HandleRPC(ctx context.Context, rs stats.RPCStats) {
-	h.handleRPC(ctx, rs)
+	h.handleRPC(ctx, rs, false)
```

Comment:
I do not find it readable.

I propose the following.

```suggestion
	isServer := false
	h.handleRPC(ctx, rs, isServer)
```